course_catalog/etl/loaders.py

```diff
@@ -203,7 +208,8 @@ def load_program(program_data):
 
 def load_programs(programs_data):
     """Load a list of programs"""
-    return [load_program(program_data) for program_data in programs_data]
+    blacklist = []  # load_course_blacklist()
```

Comment:
What is the comment for here?

Reply:
Sorry, was testing something and forgot to revert it